src/components/SubPage/SubpageTabs.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Flex, Tabs } from "antd";
+import { SubPage } from "../../types";
+import { EditingDesign } from "../../templates/disease-cell-line";
+import EditingDesignSubpage from "./EditingDesignSubpage";
+
+const {
+    container,
+    labelGroup,
+} = require("../../style/subpage-tabs.module.css");
+
+interface SubpageTabsProps {
+    tabsToRender: SubPage[];
+    editingDesignData: EditingDesign;
+}
+
+const SubpageTabs: React.FC<SubpageTabsProps> = ({
+    tabsToRender,
+    editingDesignData,
+}) => {
+    // TODO: Swap out actual components for placeholders
+    const tabComponents = {
+        [SubPage.EditingDesign]: (
+            <EditingDesignSubpage editingDesignData={editingDesignData} />
```

Comment:
```suggestion
            <EditingDesignSubpage {...editingDesignData} />
```
so that the props are still clean for the subpage tab but the editing design subpage doesn't have to unpack all the props from an object

Reply:
I also think this is better for react: I don't think it does diffs on nested prop objects (but I could be wrong)